src/test/java/com/commercetools/pspadapter/notification/processor/impl/DefaultNotificationProcessorTest.java

```diff
@@ -29,15 +28,19 @@
     Gson gson;
 
     @Test
-    public void whenNotificationHasNoProcessor_defaultProcessorIsUsed () {
+    public void whenNotificationHasNoProcessor_defaultProcessorIsUsed() {
         CtpFacade ctpFacade = mock(CtpFacade.class);
         Payment mockPayment = mock(Payment.class);
         DefaultNotificationProcessor mockDefaultProcessor = mock(DefaultNotificationProcessor.class);
         when(mockDefaultProcessor.processEventNotification(any(), any()))
                 .thenReturn(CompletableFuture.completedFuture(mockPayment));
         PaymentSaleCompletedProcessor paymentCompletedProcessor = new PaymentSaleCompletedProcessor(new GsonBuilder().create());
-        
-        NotificationProcessorContainer container = new NotificationProcessorContainerImpl(paymentCompletedProcessor, mockDefaultProcessor);
+        PaymentSaleRefundedProcessor paymentRefundedProcessor = new PaymentSaleRefundedProcessor(new GsonBuilder().create());
+        PaymentSaleDeniedProcessor paymentDeniedProcessor = new PaymentSaleDeniedProcessor(new GsonBuilder().create());
+        PaymentSaleReversedProcessor paymentReversedProcessor = new PaymentSaleReversedProcessor(new GsonBuilder().create());
```

Comment:
why we need a new instance of `Gson` every time? Is it mutable? You already have injected Gson above. Can't it be re-used?

Reply:
https://github.com/commercetools/commercetools-paypal-plus-integration/pull/64/commits/ef8689ee0d54a2ef058fc670e827c3785af7e26c